.github/workflows/validate-case-studies-structures.yaml

```diff
@@ -0,0 +1,46 @@
+name: Validate case studies structures
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - 'config/casestudies/*.yml'
+
+jobs:
+  validate:
+    if: github.event.pull_request.draft == false
```

Comment:
Can we make conditions here in such a manner that this workflow should only run when there are changes in the `/config/casestudies/` directory? @derberg @tthijm

Reply:
Line 7 already ensures that the workflow will only run if changes occurred to any file matching `config/casestudies/*.yml`.